core/search/templates/search/search_results.js

```diff
@@ -7,12 +7,24 @@ function collapseResults( group ) {
     }
     
     var results = group.find( 'ol' ).children( 'li' );
-    if ( results.length > 5 && dataModel !== openGroup ) {
-        results.hide().slice( 0, 5 ).show();
-        if (results.length > 50 && dataModel != 'Staff Directory') {
-            var numMore = 45;
+    if ( dataModel !== openGroup && (
+         ( dataModel == 'Staff Directory' && results.length > 20 ) ||
+         ( dataModel != 'Staff Directory' && results.length > 5 )
+    ) ) {
```

Comment:
You shouldn't need L:10 because of the `return` [condition on L:5](https://github.com/cfpb/collab/pull/68/files#diff-ff79fcdeb46400e96716a0cb25885f34R5). As far as the other two, I'd make it a variable like `cutOff` and assign it conditionally. Something like 

```
var cutOff = ( dataModel == 'Staff Directory' ) ? 20 : 5;

if ( results.length > cutOff ) { ... }
```